src/__tests__/develop.spec.js

```diff
@@ -0,0 +1,20 @@
+import GetNumberSum from '../develop.js'
+
+function test() {
+    const testCases = [
+      { input: { a: 1, b: 2 }, expected: 3 },
+      { input: { a: -2, b: 1 }, expected: 0 },
+      { input: { a: 'not a number', b: 'number' }, expected: NaN } 
+    ];
+  
+    const testResults = [];
+  
+    testCases.map((testCase) => {
```

Comment:
This is not the way we use map. Please see documentation about this method

Reply:
the map creates a new array for each element it accesses, forEach don't create a new array

Reply:
@Shvme7 if we want to use map there is no need to create a array and push it direclry.
We simply can return a result that we want to see as array
Because, as you said, it will rerurn an array
```suggestion
    const testResults = testCases.map((testCase) => {
```